app/app/urls.py

```diff
@@ -66,6 +66,7 @@
     url(r'^funding/embed/?', dashboard.embed.embed, name='embed'),
     url(r'^funding/avatar/?', dashboard.embed.avatar, name='avatar'),
     url(r'^profile/(.*)?', dashboard.views.profile, name='profile'),
+    url(r'^toolbox/?', dashboard.views.toolbox, name='toolbox'),
```

Comment:
can we comment this out for now?   just since i'm not exactly ready to launch the toolbox yet... so i dont want the app to serve it when merge happens